extras/rootless/containerd-rootless.sh

```diff
@@ -107,6 +117,25 @@ if [ -z $_CONTAINERD_ROOTLESS_CHILD ]; then
 			export _CONTAINERD_ROOTLESS_SELINUX
 		fi
 	fi
+
+	case "$CONTAINERD_ROOTLESS_ROOTLESSKIT_DETACH_NETNS" in
+	auto)
+		if rootlesskit --help | grep -qw -- "--detach-netns"; then
+			CONTAINERD_ROOTLESS_ROOTLESSKIT_FLAGS=--detach-netns $CONTAINERD_ROOTLESS_ROOTLESSKIT_FLAGS
```

Comment:
I do not think this is doing what you think.

What will happen instead is:

```
VAR=VALUE command
```

Will fix.

Reply:
Fixed in https://github.com/containerd/nerdctl/pull/3214 (thanks)